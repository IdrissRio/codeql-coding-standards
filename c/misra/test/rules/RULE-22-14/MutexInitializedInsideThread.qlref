rules/RULE-22-14/MutexInitializedInsideThread.ql