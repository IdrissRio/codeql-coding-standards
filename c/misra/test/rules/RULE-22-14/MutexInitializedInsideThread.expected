| test.c:110:3:110:10 | call to mtx_init | Mutex initialization reachable from threaded function '$@'. | test.c:105:6:105:32 | from_root7_use_thread_local | from_root7_use_thread_local |
